code/modules/mod/mod_control.dm

```diff
@@ -633,6 +608,21 @@
 	balloon_alert(wearer, "no power!")
 	toggle_activate(wearer, force_deactivate = TRUE)
 
+/obj/item/mod/control/proc/set_mod_color(new_color)
+	var/list/all_parts = mod_parts.Copy() + src
+	for(var/obj/item/part as anything in all_parts)
+		part.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
```

Comment:
Was this meant to be FIXED_COLOUR_PRIORITY ? If so that's unnecessary since add_colour already overrides it ?